runtime/ops/manipulation.cc

```diff
@@ -8,8 +8,8 @@
 namespace chainer_compiler {
 namespace runtime {
 
-chainerx::Array ShapeOp::RunImpl(ChxVMState* st, const chainerx::Array& data) {
-    return ShapeToArray(data.shape());
+void ShapeOp::RunImpl(ChxVMState* st) {
```

Comment:
I want the code generator to generate the prototype like `chainerx::Shape ShapeOp::RunImpl(ChxVMState* st, const chainerx::Array& data);` and returned shape will be automatically `SetShape`-ed.